### PR TITLE
Update configuration to use new messages model

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
Related to: [uw-frame#484](https://github.com/UW-Madison-DoIT/uw-frame/pull/484)
Related to: [angularjs-portal overlay updates](https://git.doit.wisc.edu/myuw-overlay/angularjs-portal/merge_requests)

**In this PR:**
- Update override paths to use new app-config
- Update static messages source

----

Contributor License Agreement adherence:

- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: angularjs-portal-home/src/main/webapp/js/override.js [hunk 1]
```diff
@@ -28,11 +25,17 @@ define(['angular'], function(angular) {
...
             },
-            'notificationsURL': '/web/staticFeeds/notifications.json',
+            'messagesURL': '/web/staticFeeds/messages.json',
             'groupURL': '/portal/api/groups',
             'kvURL': '/storage',
             'loginSilentURL': '/portal/Login?silent=true&profile=bucky',
           },
+          'MESSAGES': {
+            'announcementsEnabled': true,
+            'notificationsEnabled': true,
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/js/override.js]: Do we need these flags, or could the enablement of announcements and notifications be implied by whether there are announcement-flavored and notification-flavored items in the messages feed? If there's no announcements in the feed, then they're not enabled. If there's no notifications in the feed, then they're not enabled.

File: angularjs-portal-home/src/main/webapp/js/override.js [hunk 2]
```diff
@@ -28,11 +25,17 @@ define(['angular'], function(angular) {
...
+            'groupFiltering': true,
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/js/override.js]: Do we need this flag, or could group filtering be implied by whether there are groups in the items in the messages feed? If groups are specified, filter to them. If they're not, don't worry about it.

Comment (thevoiceofzeke) [inline angularjs-portal-home/src/main/webapp/js/override.js]: ¯\_(ツ)_/¯

Valid questions, but I don't feel confident that the decision to remove the flags should be up to me. As it is, I've just repurposed what was already there.

**Comment by ChristianMurphy** [comment]
@thevoiceofzeke CI failure is coming for RemarkLint.
Cherry-picking or applying change from 372b343840662e8688f2fa03fb0845990e1c6b9a should resolve.

**Pull request merged**